bin/genEvalSpecializations.py

```diff
@@ -374,7 +374,7 @@ class Evaluation<ValueT, {{ numDerivs }}>
     }
 
     template <class RhsValueType>
-    static Evaluation createVariable(const Evaluation& x, const RhsValueType& value, int varPos)
```

Comment:
like with OPM/ewoms#523, please silence such warnings using the OPM_UNUSED macros. (I will not mention this from here on, but please fix the other signatures as well.)